### ISSUE TITLE
ProtobufOutputFormatter error with large integers

### ISSUE THREAD

**Issue by Exocomp** [WebApiContrib/WebAPIContrib.Core#183]
**Summary**: ProtobufOutputFormatter modifies the original bytes generated by protobuf.net resulting in decode error with protobufjs.

**Server**: WebApiContrib.Core.Formatter.Protobuf 2.1.2
**Client**: Angular 8.1.0, protobufjs 6.8.8

**C# Class**:

```
[ProtoContract]
public class ExampleClass
{
	[ProtoMember(1)]
	public int ExampleInt { get; set; }
}
```

**Proto File**:
```
// model.proto
syntax = "proto3";
 
package model;
 
message ExampleClass {
    optional int32 exampleInt = 1;
}
```

**Object to Encode**:

```
var returnObj = new ExampleClass
{
	ExampleInt = 123456789,
};
```

When the above object is encoded by protobuf.net it results with the following bytes:

`8, 149, 154, 239, 58`

However 'WebApiContrib.Core.Formatter.Protobuf' returns the following bytes:

`8, 239, 191, 189, 239, 191, 189, 239, 191, 189, 58`

As a result on the client side protobufjs is not able to decode it correctly. It returns the following on the client side:

`{exampleInt: -34643985}`

Note how the original value was "123456789" but decoded to "-34643985".

What am I missing? I'd appreciate pointing me in the right direction.

**Comment by damienbod** [comment]
Hi @Exocomp This is a bug in version 2.1.2

https://stackoverflow.com/questions/2152978/using-protobuf-net-i-suddenly-got-an-exception-about-an-unknown-wire-type

I will fix this a s soon as possible. As a workaround, revert to version 2.1.1

**Comment by damienbod** [comment]
fixed now in version 2.1.3
